cache/lru_cache.cc

```diff
@@ -513,6 +518,20 @@ size_t LRUCache::TEST_GetLRUSize() {
   return lru_size_of_all_shards;
 }
 
+double LRUCache::TEST_GetHighPriPoolRatio() {
```

Comment:
Remove the TEST_ prefix for both of the methods?

Reply:
Thank you for the review. I made the required changes and merged with the latest master. Could you please take a look. Thank you.